randovania/layout/description_migration.py

```diff
@@ -615,6 +615,19 @@ def _migrate_v32(data: dict) -> None:
                     region_location[new_area_name] = region_location.pop(old_area_name)
 
 
+def _migrate_v33(data: dict) -> None:
+    game_modifications = data["game_modifications"]
+
+    for game in game_modifications:
+        if game["game"] == "prime2":
+            banned_pickups = ["Cannon Ball", "Unlimited Beam Ammo", "Unlimited Missiles", "Double Damage"]
+            if "pickups" in game["starting_equipment"]:
+                starting_pickups = game["starting_equipment"]["pickups"]
```

Comment:
@henriquegemignani Couldn't this also be in `game["starting_equipment"]["items"]` from a very old rdvgame due to _migrate_v14?

Reply:
Yes, but then it's not pickups it's items directly and don't think it'll give an error with the new code

Reply:
Don't know if understand correctly (maybe because I don't even know why it needs to be banned). Throwing no error is always a good thing but don't you need to remove it from the `items` in `starting_equiment` in that case?

Reply:
It's perfectly fine to have it be a starting item, it does literally nothing.

Reply:
Ok, then it's fine how it is I guess.